### PR TITLE
Enable sentry logging 

### PR DESCRIPTION
**Pull request by sbose78**
to populate https://errortracking.prod-preview.openshift.io/openshift_io/fabric8-auth/ whenever there's a 500

Based on associated changes in https://github.com/fabric8-services/fabric8-wit/tree/master/sentry

configmap changes https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/1785

### CODE REVIEW

File: openshift/auth.config.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ objects:
     oso.regapp.admin.username: Cg==
     oso.regapp.admin.token: Cg==
     oso.regapp.url: Cg==
+    environment: prod-preview
```
Comment (alexeykazakov) [inline openshift/auth.config.yaml]: Move it to ConfigMap

File: sentry/sentry.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	q := *m
+	token := goajwt.ContextJWT(ctx)
+	if token == nil {
+		return nil, fmt.Errorf("no token found in context")
+	}
+	t, err := q.ParseToken(ctx, token.Raw)
```
Comment (alexeykazakov) [inline sentry/sentry.go]: Or just `t, err := m.ParseToken(ctx, token.Raw)` No need to define q.

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-460-3`

**Comment by sbose78** [comment]
@alexeykazakov  In this PR, I propose setting SENTRY_DSN as AUTH_SENTRY_DSN - not sure why we didn't do it ( here and in WIT). It feels inconsistent to me otherwise.

**Comment by alexeykazakov** [comment]
SENTRY_DSN is used by the default Sentry client. But WIT doesn't use the default client (and your PR doesn't use it either). So, we should be fine with AUTH_SENTRY_DSN but it's worth testing.

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-460-5`

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-460-6`

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-460-8`

**Pull request merged**